tutorials/subdomain_and_interface_projections.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>First sentence: I would make the (obvious when you know it) remark that the non-matching grid is needed to see the difference between intensive and extensive quantities. Or a different comment to that effect.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>